### PR TITLE
doc/uftrace.md: Add a common list of all options (generated from -h)

### PR DESCRIPTION
**Pull request by bernhardkaindl**
While they are described in the command-specific pages, it would be nice to also have one place to check for all options.

### CODE REVIEW

File: doc/uftrace.md [hunk 1]
```diff
@@ -112,6 +124,266 @@ OPTIONS
...
+:   Use *FORMAT* for output: normal, html (default: normal)
+
+-f, \--output-fields=*FIELD*
+:   Show FIELDs in the replay or graph output
+
+-F, \--filter=*FUNC*
+:   Only trace those FUNCs
+
+-g
+:   \--agent                Start an agent in mcount to listen to commands
```
Comment (namhyung) [inline doc/uftrace.md:211]: A broken format, please update.

File: doc/uftrace.md [hunk 2]
```diff
@@ -66,9 +66,21 @@ script
...
+[//2]: # "https://stackoverflow.com/questions/4823468/comments-in-markdown"
+
+[//3]: # "Most of the command line option help texts have been generated by:"
+[//4]: # "https://github.com/bernhardkaindl/help2md"
+[//5]: # "(followed by merging the existing more descriptive option help texts)"
+[//6]: # "TODO: Group them into command-specific sections for a quick overview"
+[//7]: # "with a note referring to the command-specific page for each section."
+[//8]: # "While they are described in the command-specific pages, it"
+[//9]: # "would be nice to also have one place to check for all options."
+
```
Comment (namhyung) [inline doc/uftrace.md:79]: Two blank lines between sections please.

File: doc/uftrace.md [hunk 3]
```diff
@@ -112,6 +124,266 @@ OPTIONS
...
+-W, \--watch=*POINT*
+:   Watch and report *POINT* if it's changed
+
+-Z, \--size-filter=*SIZE*
+:   Apply dynamic patching for functions bigger than *SIZE*
+
+For more detail about these command-specific options,
+please see the more specific manual pages listed below.
+
```
Comment (namhyung) [inline doc/uftrace.md:386]: Ditto, it needs one more blank line.

File: doc/uftrace.md (hunk 1 above)
Comment (bernhardkaindl) [reply doc/uftrace.md]: Thanks, fixed!

**Review by namhyung — APPROVED** [review]
LGTM

**Pull request merged**